indexer/packages/postgres/src/stores/wallet-table.ts

```diff
@@ -123,7 +124,7 @@ export async function updateTotalVolume(
   windowEndTs: string,
 ) : Promise<void> {
 
-  await knexReadReplica.getConnection().raw(
+  await knexPrimary.raw(
```

Comment:
after reviewing some of this, I'm being more convinced that we should fix the nits in the previous PRs for three reasons:
- As I'm reviewing a PR to `add update affiliate info into roundtable`, there are a bunch of unrelated changes that are confusing as a reader
- Reviewing this PR now requires context from multiple previous PRs, and is impossible to track.

I would prefer that a change like this is completely separated from affiliate info roundtable. This is real business logic changes that if caused a bug would be difficult to track when reviewing PR history. Ie why would adding updates to affiliate info roundtable change the behavior of updateTotalVolume?

Reply:
But im ok with just addressing everything in one PR as well